how-to-guides/hello-neptune/notebooks/hello_neptune.ipynb

```diff
@@ -101,35 +101,13 @@
     ]
    },
    "source": [
-    "## Initialize Neptune and create new run\n",
+    "## Initialize Neptune and create a new run\n",
     "\n",
-    "To create a new run for tracking the metadata, you tell Neptune who you are (`api_token`) and where to send the data (`project`).\n",
+    "To create a new run for tracking the metadata, you must tell Neptune who you are (`api_token`) and where to send the data (`project`). If you haven't registered, you can create an anonymous run in a public project. \n",
     "\n",
-    "You can use the default code cell below to create an anonymous run in a public project. **Note**: Public projects are cleaned regularly, so anonymous runs are only stored temporarily.\n",
+    "**Note**: Public projects are cleaned regularly, so anonymous runs are only stored temporarily.\n",
     "\n",
-    "### Log to your own project instead\n",
-    "\n",
-    "Replace the code below with the following:\n",
-    "\n",
-    "```python\n",
-    "import neptune\n",
-    "from getpass import getpass\n",
-    "\n",
-    "run = neptune.init_run(\n",
-    "    project=\"workspace-name/project-name\",  # replace with your own (see instructions below)\n",
-    "    api_token=getpass(\"Enter your Neptune API token: \"),\n",
-    ")\n",
-    "```\n",
-    "\n",
-    "To find your API token and full project name:\n",
-    "\n",
-    "1. [Log in to Neptune](https://app.neptune.ai/).\n",
-    "1. In the bottom-left corner, expand your user menu and select **Get your API token**.\n",
-    "1. The workspace name is displayed in the top-left corner of the app. \n",
-    "\n",
-    "    To copy the project path, in the top-right corner, open the settings menu and select **Properties**.\n",
-    "\n",
-    "For more help, see [Setting Neptune credentials](https://docs.neptune.ai/setup/setting_credentials) in the Neptune docs."
```

Comment:
Shouldn't we inform users of this option before they create an ANONYMOUS run in the public project?
Otherwise they'll just execute this cell and create an ANONYMOUS run, before realising they could have used their own credentials

Reply:
Well, it's stated in the beginning that the purpose of this notebook is to try Neptune as an anonymous user. The intro also explains that it might work for you if you haven't registered. The info on using your own credentials is additional and moved into its own section.

Reply:
But we also tell them at the beginning that they can register and create their own project. Flow-wise, it is off as even if someone has already registered and created a project, they still won't know where to enter their personal details while initializing a run until after they've already created an anonymous run.

Reply:
@szaganek Since we're digging into this part now, let's in parallel update our Notebook template. It's overdue for a review and it's already in our backlog: https://app.clickup.com/t/869492mk8